.coveragerc

```diff
@@ -0,0 +1,6 @@
+[run]
+omit =
+    */__init__.py
```

Comment:
Is this here to avoid checking non-empty __init__.py where you are just importing things from the other modules to expose them at the package level?

Does this format support comments? if so it would be nice to have it as comment.

Reply:
Does not support